plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/projecttype/PlainJavaValueProviderFactory.java

```diff
@@ -91,14 +97,15 @@ public void setValues(String attributeName, List<String> values) throws ValueSto
     private List<String> getOutputFolder() throws ValueStorageException {
       String outputDir;
       try {
-        outputDir = extensionService.getOutputDir(wsPath);
+        outputDir = extensionService.getOutputDir(prefixURI(wsPath));
```

Comment:
Don`t add prefix for `wsPath`. It will be added inside `getOutDir` method.

Reply:
I think it should be noted in `getOutputDir` javadoc

Reply:
fixed